homeassistant/components/honeygain/__init__.py

```diff
@@ -0,0 +1,86 @@
+"""The Honeygain integration."""
+from __future__ import annotations
+
+from datetime import timedelta
+
+from pyHoneygain import HoneyGain
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_EMAIL, CONF_PASSWORD, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.util import Throttle
+
+from .const import DOMAIN, LOGGER, UPDATE_INTERVAL_MINS
+
+PLATFORMS: list[Platform] = [Platform.SENSOR, Platform.BUTTON]
```

Comment:
New integrations allow only one platform to be added to make the PR as small as possible. It is also not recommended to add new features to an open PR.

I think the you should focus on one platform and open a follow-up PR for each other PR.

Reply:
Ahh yes, I remember seeing this on the documentation, I'll remove the button now and leave this as a sensor integration for now